### DISCUSSION TITLE
Open VDB Files not rendering

### DISCUSSION THREAD

**Discussion by PARTHA313** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#488] [General]
![image](https://user-images.githubusercontent.com/104050993/164150565-9235f274-0f2e-480a-b9c3-ddd28dc9d431.png)
![image](https://user-images.githubusercontent.com/104050993/164150585-f093e627-32f8-46dd-b025-6e0601839506.png)
Please help me.

**Comment by bsavery** [comment]
Please open an issue and include a scene.

**Comment by bsavery** [comment]
Ah ok I see you did!  Thanks
